src/WalterModem.h

```diff
@@ -819,6 +819,29 @@ typedef enum {
     WALTER_MODEM_COAP_OPT_VALUE_APPLICATION_CBOR = 60
 } WalterModemCoapOptValue;
 
+/**
+ * @brief Enum containing the mqtt response codes
+ */
+typedef enum {
```

Comment:
Please add explicit values, as this better reflects the AT reference manual of the Sequans GM02SP.